pennylane/vqe/vqe.py

```diff
@@ -158,20 +158,26 @@ def simplify(self):
         self._ops = ops
 
     def __str__(self):
-        terms = []
+        # lambda function that formats the wires
+        wires_print = lambda ob: ",".join(map(str, ob.wires.tolist()))
```

Comment:
Cool! So `,` would be placed in between wires, correct? Maybe worth adding a Hermitian print example to the docstring of the class or having a small example in another comment above

Reply:
I will add some detail to the docstring showing how we could have a general operation.

Reply:
Done.  
Now `,` is switched for `'`.  Only gut feeling for the change, but it seems to remove my feelings of uncomfortableness around `op0,1` to have `op0'1`